R/html_dependencies.R

```diff
@@ -55,9 +60,59 @@ html_dependency_bootstrap <- function(theme) {
       # These shims are necessary for IE 8 compatibility
       "shim/html5shiv.min.js",
       "shim/respond.min.js"),
-    stylesheet = paste0("css/", theme, ".min.css"))
+    stylesheet = paste0("css/", theme, ".min.css"),
+    head = format(tags$style(HTML(
+      "h1 {font-size: 34px;}
+       h1.title {font-size: 38px;}
+       h2 {font-size: 30px;}
+       h3 {font-size: 24px;}
+       h4 {font-size: 18px;}
+       h5 {font-size: 16px;}
+       h6 {font-size: 12px;}"
+    )))
```

Comment:
Probably not worth moving this from html template because shinytest records this